src/main/kotlin/com/rose/gateway/minecraft/logging/FileLogger.kt

```diff
@@ -0,0 +1,28 @@
+package com.rose.gateway.minecraft.logging
+
+import com.rose.gateway.GatewayPlugin
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+import java.util.logging.FileHandler
+import java.util.logging.Logger
+
+/**
+ * Helper object that provides logging to file
+ */
+object FileLogger : KoinComponent, Logger("File Logger", null) {
```

Comment:
There are two options here. First, you can remove the inheritance from FileLogger, since you only ever use the internal logger anyways. Second, you can remove the internal logger and add the handlers to this object directly. Using both is redundant